rust/lance-index/src/vector/ivf/shuffler.rs

```diff
@@ -452,7 +452,12 @@ impl IvfShuffler {
                 )?;
 
                 for batches in shuffled {
-                    file_writer.write(&batches).await?;
+                    if batches.is_empty() {
+                        continue;
+                    }
+                    // concat to avoid slowdown due to small IOs
+                    let batch = concat_batches(&batches[0].schema(), &batches)?;
+                    file_writer.write(&[batch]).await?;
```

Comment:
Is this still an issue for v2 ? @westonpace

Reply:
huh I'm kind of shocked it's an issue for v1 in the first place.